internal/controller/monitor/monitor.go

```diff
@@ -126,6 +126,23 @@ func (m *Monitor) getScheduledCommandJobs(ctx context.Context, queue string) (jo
 		agentQueryRule = append(agentQueryRule, fmt.Sprintf("queue=%s", queue))
 	}
 
+	queues, err := api.GetClusterQueues(ctx, m.gql, m.cfg.Org, m.cfg.ClusterUUID, 100)
```

Comment:
I think it may be worth a TODO comment to change this approach once there is a way to get info for a particular queue in a targeted query

```suggestion
	// TODO: use a more targeted query once one becomes available
	queues, err := api.GetClusterQueues(ctx, m.gql, m.cfg.Org, m.cfg.ClusterUUID, 100)
```